.github/workflows/canary-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy a Canary Build
+on:
+  workflow_dispatch:
+    branches-ignore:
+      - master
+jobs:
+  deploy_canary:
+    name: Build And Deploy Canary Release
+    runs-on: ubuntu-latest
```

Comment:
Add `environment: canary` here. This you, Ken, Jake and I to approve canary deployments before they go out.

Reply:
I think what I've added is correct here. Unfortunately we can't check until this is merged. 😆